### PR TITLE
fix: add favicons

### PR DESCRIPTION
**Pull request by Ja7ad**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,3 +20,4 @@ tmp
 
 # vscode
 .vscode
+Gemfile.lock
```
Comment (b00f) [inline .gitignore:23]: Let's keep `Gemfile.lock` as the version in GitHub Actions changes, and we may fail to compile the website.

File: assets/favicons/site.webmanifest [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+    "name": "",
```
Comment (b00f) [inline assets/favicons/site.webmanifest:2]: `safari-pinned-tab` is not same as other icons.

**Review by b00f — CHANGES_REQUESTED** [review]
Why don't use PIP icon:

https://github.com/pactus-project/PIPs/blob/main/assets/readme/pip.png

**Comment by Ja7ad** [comment]
@b00f Done

**Review by b00f — APPROVED** [review]

**Pull request merged**